### ISSUE TITLE
Update PWA manifest "shortcuts" with the last few games added to Stadia Pro

### ISSUE THREAD

**Issue by jeremyBanks** [stadians/stadians#27]
continuing from #26 

The PWA `manifest.json` allows us to specify a list of shortcuts, which may be made available as menu items. This seems particularly neat on mobile.

![](https://user-images.githubusercontent.com/18020/93039427-71126b00-f615-11ea-923b-130448ea179b.png)

There doesn't seem to be any way to explicitly update the stored version of the manifest, but it appears to only be cached for a few days. So perhaps the most useful things we could add to that would be the last few games added to Stadia Pro.

**Comment by jeremyBanks** [comment]
Done. Also, I ditched my iPhone and am on Android now. 🙃 

<img width="320" src="https://user-images.githubusercontent.com/18020/93411736-d52d6d00-f869-11ea-93ea-fdd5f2bb3e2d.jpg" />


However, to make the links work properly, I had to get rid of the server-side redirects (#5) and do them client-side, since the PWA doesn't open the Stadia app if you load it directly the redirect. It still looks smooth, so that's fine.
